index.html

```diff
@@ -244,6 +245,115 @@ <h3 class="project1_title">Multi-Post Stories</h3>
+                        </div>
+                    </div>
+                    <ul class="tonic_list">
+                        <li class="popup_canopy_desktop canopy"></li>
+                        <li> <img class="counter" src="img/Counter.png" alt="counter" /> </li>
+                        <li class="popup_li1_desktop"></li>
+                        <li> <img class="counter" src="img/Counter.png" alt="counter" /></li>
+                        <li class="popup_li2_desktop"></li>
+                    </ul>
+                    <div class="project1_img">
+                        <img src="1" alt="multi-post-stories" class="popup_img_desktop" />
+                    </div>
+
+                    <div class="desktop_popup_positioner">
+                        <div class="popup_description_desktop_container">
+                            <p class="popup_description_desktop"></p>
+                        </div>
+
+                        <div>
+                            <div class="small_button_project1_popup">
+                                <ul class="button_list">
+                                    <li class="language_descriptor language_descriptor1_desktop"></li>
+                                    <li class="language_descriptor language_descriptor2_desktop"></li>
+                                    <li class="language_descriptor
+                                language_descriptor3_desktop"></li>
+                                </ul>
+                                <ul class="button_list">
+                                    <li class="language_descriptor
+                                language_descriptor4_desktop"></li>
+                                    <li class="language_descriptor
+                                language_descriptor5_desktop"></li>
+                                    <li class="language_descriptor
+                                language_descriptor6_desktop"></li>
+                                </ul>
+                            </div>
+
+                            <img src="img/mobile_popup_seperator.png" alt="seperator" class="mobile_popup_seperator" />
+                            <div class="project1_popup_buttons">
+                                <div class="see_live_container">
+                                    <button class="see_project_mobile_popup1" type="button">See live <img
+                                            src="img/see_live.png" alt="see_live_icon"></button>
+                                </div>
+                                <div class="see_source_container">
+                                    <button class="see_project_mobile_popup2" type="button">See Source <img
+                                            src="img/see_source.png" alt="github_logo"></button>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </aside>
+
+
+
```

Comment:
- [ ] As I mentioned in the previous comment, I can suggest you implement one template of the Popup window in your JS file (instead of having duplicate code as in this case) then make sure the Modal popup data is populated with the data from the projects array as required for the project.